src/modules/launcher/Microsoft.Launcher/dllmain.cpp

```diff
@@ -364,15 +364,30 @@ void Microsoft_Launcher::parse_hotkey(PowerToysSettings::PowerToyValues& setting
 {
     try
     {
-        auto jsonHotkeyObject = settings.get_raw_json().GetNamedObject(JSON_KEY_PROPERTIES).GetNamedObject(JSON_KEY_OPEN_POWERLAUNCHER);
-        m_hotkey.win = jsonHotkeyObject.GetNamedBoolean(JSON_KEY_WIN);
-        m_hotkey.alt = jsonHotkeyObject.GetNamedBoolean(JSON_KEY_ALT);
-        m_hotkey.shift = jsonHotkeyObject.GetNamedBoolean(JSON_KEY_SHIFT);
-        m_hotkey.ctrl = jsonHotkeyObject.GetNamedBoolean(JSON_KEY_CTRL);
-        m_hotkey.key = static_cast<unsigned char>(jsonHotkeyObject.GetNamedNumber(JSON_KEY_CODE));
+        auto settingsObject = settings.get_raw_json();
+
+        if (settingsObject.HasKey(JSON_KEY_PROPERTIES))
+        {
+            auto jsonHotkeyObject = settingsObject.GetNamedObject(JSON_KEY_PROPERTIES).GetNamedObject(JSON_KEY_OPEN_POWERLAUNCHER);
+            m_hotkey.win = jsonHotkeyObject.GetNamedBoolean(JSON_KEY_WIN);
+            m_hotkey.alt = jsonHotkeyObject.GetNamedBoolean(JSON_KEY_ALT);
+            m_hotkey.shift = jsonHotkeyObject.GetNamedBoolean(JSON_KEY_SHIFT);
+            m_hotkey.ctrl = jsonHotkeyObject.GetNamedBoolean(JSON_KEY_CTRL);
+            m_hotkey.key = static_cast<unsigned char>(jsonHotkeyObject.GetNamedNumber(JSON_KEY_CODE));
+        }
+        else
+        {
+            Logger::info("PT Run start shortcut is not present in settings. Use default shortcut instead.");
+            m_hotkey.win = false;
+            m_hotkey.alt = true;
+            m_hotkey.shift = false;
+            m_hotkey.ctrl = false;
+            m_hotkey.key = VK_SPACE;
+        }
     }
     catch (...)
     {
+        Logger::error("Failed to initialize PT Run start shortcut");
```

Comment:
As suggested by @ivan100sic , let's refactor the code and move the code, that now is in the "else", out of the try catch.
If `m_hotkey` is not set in the try-catch, we will set it to its default value.

Reply:
done